src/main/java/com/artipie/rpm/ModifiableRepository.java

```diff
@@ -97,9 +97,12 @@ public ModifiableRepository update(final FilePackage pkg) throws IOException {
      * Clears records about packages that does not present in the repository any more
      * from metadata files.
      * @return Itself
+     * @throws IOException On error
      */
-    public ModifiableRepository clear() {
-        this.metadata.forEach(item -> item.clear(this.existing));
+    public ModifiableRepository clear() throws IOException {
+        for (final Metadata item : this.metadata) {
```

Comment:
@olenagerasimova does this metadata items shares some common state? If not, I'd use:
```java
this.metadata().stream().parallel().forEach(Metadata::brash)
```
Parallel update for metadata items increases performance of RPM more than twice.